targets/runtimeDependencies/project.json

```diff
@@ -3,7 +3,9 @@
     "win7-x86": {},
     "win7-x64": {},
     "osx.10.10-x64": {},
-    "ubuntu.14.04-x64": {}
+    "ubuntu.14.04-x64": {},
+    "ubuntu.16.04-x64": {},
+    "ubuntu.16.10-x64": {}
```

Comment:
_Do_ we support `16.10`? I thought we were just going for LTS releases, but I see 16.10 on corefx, too.